scripts-dev/strip_returnvalue.py

```diff
@@ -0,0 +1,21 @@
+import re
```

Comment:
this doesn't feel worth committing?

Reply:
if some later merges have code using it i can just scrounge it from my hdd I guess :)